models/account.py

```diff
@@ -44,7 +50,7 @@ def __init__(self, message="Sorry, there was a problem, please try later"):
 class AccountMethods(Account):
     @staticmethod
     def get_account(
-        email: str = None, account_id: str = None
+        email: str = None, account_id: str = None, azure_ad_subject_id = None
     ) -> Account | None:
         """
         Get an account from the account store using either
```

Comment:
could account error and account methods classess not be classess and just methods all under the account class?would that be better/ more modular?